loaders/loaders.go

```diff
@@ -265,6 +267,25 @@ type Server struct {
 	KOp            *babyjub.PublicKey
 }
 
+func (s *Server) Start() error {
+	go func() {
+		log.Info("Starting Issuer Server")
+		for {
+			select {
+			case <-time.After(s.Cfg.Issuer.PublishStatePeriod):
+				if err := s.Issuer.PublishState(); err != nil {
+					log.Error(err)
+				}
+			case <-time.After(s.Cfg.Issuer.SyncIdenStatePublicPeriod):
+				if err := s.Issuer.SyncIdenStatePublic(); err != nil {
+					log.Error(err)
+				}
+			}
+		}
+	}()
+	return nil
+}
+
```

Comment:
Please, add a Stop / StopAndJoin function.

Something like this: https://github.com/iden3/go-iden3-core/blob/8ac92eef7b5c7fb1b2e1921a273097a601765fac/services/rootsrv/service.go#L130